.github/workflows/demo-list-branches.yml

```diff
@@ -0,0 +1,30 @@
+name: List branches and run matrix job
+
+on:
+  workflow_dispatch:
+
+jobs:
+  list-branches:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: List branches on repository
+        uses: bonitasoft/actions/packages/list-branches@main
+        with:
+          branche_regex: '^main$|test-flow'
```

Comment:
💡 Suggestion

Something like `selected_branches` and make it accept an array like this:
```yaml
 name: List branches on repository
        uses: bonitasoft/actions/packages/list-branches@main
        with:
          selected_branches: 
            - main
            - release-*
```